README.md

```diff
@@ -105,38 +105,50 @@ Codecov's Action supports inputs from the user. These inputs, along with their d
-| `codecov_yml_path` | Specify the path to the Codecov YML | Optional
-| `commit_parent` | Override to specify the parent commit SHA | Optional
-| `directory` | Directory to search for coverage reports. | Optional
-| `disable_search` | Disable search for coverage files. This is helpful when specifying what files you want to upload with the --file option. | Optional
-| `disable_file_fixes` | Disable file fixes to ignore common lines from coverage (e.g. blank lines or empty brackets) | Optional
+| `binary` | The file location of a pre-downloaded version of the CLI. If specified, integrity checking will be bypassed. | Optional
+| `codecov_yml_path` | The location of the codecov.yml file. This is crrently ONLY used for automated test selection (https://docs.codecov.com/docs/getting-started-with-ats). Note that for all other cases, the Codecov yaml will need to be locate
+d as described here: https://docs.codecov.com/docs/codecov-yaml#can-i-name-the-file-codecovyml | Optional
+| `commit_parent` | SHA (with 40 chars) of what should be the parent of this commit. | Optional
+| `directory` | Folder to search for coverage files. Default to the current working directory | Optional
+| `disable_file_fixes` | Disable file fixes to ignore common lines from coverage (e.g. blank lines or empty brackets). Read more here https://docs.codecov.com/docs/fixing-reports | Optional
+| `disable_search` | Disable search for coverage files. This is helpful when specifying what files you want to upload with the files option. | Optional
+| `disable_safe_directory` | Disable setting safe directory. Set to true to disable. | Optional
 | `dry_run` | Don't upload files to Codecov | Optional
 | `env_vars` | Environment variables to tag the upload with (e.g. PYTHON \| OS,PYTHON) | Optional
-| `exclude` | Folders to exclude from search | Optional
-| `fail_ci_if_error` | Specify whether or not CI build should fail if Codecov runs into an error during upload | Optional
-| `file` | Path to coverage file to upload | Optional
-| `files` | Comma-separated list of files to upload | Optional
-| `flags` | Flag upload to group coverage metrics (e.g. unittests \| integration \| ui,chrome) | Optional
-| `handle_no_reports_found` | Raise no exceptions when no coverage reports found | Optional
-| `job_code` | The job code | Optional
-| `name` | User defined upload name. Visible in Codecov UI | Optional
-| `os` | Override the assumed OS. Options are linux \| macos \| windows \| . | Optional
-| `override_branch` | Specify the branch name | Optional
-| `override_build` | Specify the build number | Optional
+| `exclude` | Comma-separated list of folders to exclude from search. | Optional
+| `fail_ci_if_error` | On error, exit with non-zero code | Optional
+| `files` | Comma-separated list of explicit files to upload. These will be added to the coverage files found for upload. If you wish to only upload the specified files, please consider using disable-search to disable uploading other files. | Op
```

Comment:
Maybe `explicit list`? Otherwise I'm concerned about the type of files being uploaded to us ;)

Reply:
```suggestion
| `files` | Comma-separated explicit list of files to upload. These will be added to the coverage files found for upload. If you wish to only upload the specified files, please consider using "disable-search" to disable uploading other files. | Optional
```